ballerina/tests/test.bal

```diff
@@ -0,0 +1,316 @@
+    test:assertNotEquals(response.length(), 0, msg = "No Favourite Filters.");
+} else {
+    test:assertFail(msg = "Error occurred while fetching favourite filters: " + response.toString());
+}
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetMyFilters() returns error? {
+    // Call the get function without any query parameters
+    Filter[]|error response = jira->/rest/api/'3/filter/my();
+
+    // Assert that response is an array of Filter objects and check the array is not empty
+    test:assertTrue(response is Filter[], msg = "Expected response to be an array of Filter objects.");
+    if response is Filter[] {
+        test:assertNotEquals(response.length(), 0, msg = "There are no My Filters.");
+    }
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testPostGroup() returns error? {
+    
+
+    // Call the function to create a group
+    Group|error response = jira->/rest/api/'3/group.post(payload = {name: "first group"});
+
+    // Validate the response
+    test:assertTrue(response is Group, msg = "Expected response to be a Group object.");
+    if response is Group {
+        test:assertEquals(response.name, "first group", msg = "Group name does not match the expected value.");
+    }
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testDeleteGroup() returns error? {
+
+    // Call the function to delete the specified group
+    http:Response|error response = jira->/rest/api/'3/group.delete(queries={groupname: "first group"});
+
+    // Validate the response
+    test:assertTrue(response is http:Response, msg = "Expected a HTTP Response object.");
+    if response is http:Response {
+        test:assertEquals(response.statusCode, 200, msg = "Expected HTTP status code 200 indicating successful deletion.");
+    }
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetAllFieldConfigurations() returns error? {
+   
+    // Call the function to get all field configurations
+    PageBeanFieldConfigurationDetails|error response = jira->/rest/api/'3/fieldconfiguration();
+
+    // Validate the response
+    if response is PageBeanFieldConfigurationDetails {
+        test:assertTrue(response.total > 0, msg = "There is no field configuration");
+    }
+    //for mock validation
+    if response is json {
+        test:assertTrue(response.total > 0, msg = "There is no field configuration");
+    }
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetAllFields() returns error? {
+
+    // Call the function to get all fields
+    FieldDetails[]|error response = jira->/rest/api/'3/'field();
+
+    // Validate the response
+    if response is FieldDetails[] {
+        test:assertNotEquals(response.length(), 0, msg = "Expected field details array to be non-empty.");
+    } else {
+        test:assertFail(msg = "Expected a FieldDetails[] response.");
+    }
```

Comment:
same as above. Please check other places as well.

Reply:
@Bhashinee Done. Did all the changes to test.bal